### ISSUE TITLE
not Deployed on localhost

### ISSUE THREAD

**Issue by sonu4goods** [shipanmazumder/permission#1]
Please write some notes how we deploy this project on localhost .its giving manyerrors.

**Comment by shipanmazumder** [comment]
Please Check now..

**Comment by sonu4goods** [comment]
same issue bro user management section not found anywhere

**Comment by shipanmazumder** [comment]
app/http/controllers/permission/UserController
resource/view/admin/permission

**Comment by sonu4goods** [comment]
how module is working.can assign role to user ?

**Comment by shipanmazumder** [comment]
yes. When you create a user then you assign a role for the user.

**Comment by sonu4goods** [comment]
user module is not working in dashboard that the question from start.

**Comment by shipanmazumder** [comment]
Please Flow step by step. 
First clone this project.
then create an env file.
then run composer update
then run php artisan migrate
change the default login from to admin.login.login
change the default HomeController to DashboardController
and finally
localhost/permission/public/

**Comment by sonu4goods** [comment]
![error](https://user-images.githubusercontent.com/43901223/75113756-372a4900-5672-11ea-8186-16d8a741dfbb.png)
i am now admin dashboard.where is the use section can you please let me know

**Comment by shipanmazumder** [comment]
please click Administrator

**Comment by sonu4goods** [comment]
![module](https://user-images.githubusercontent.com/43901223/75113897-a2c0e600-5673-11ea-9e39-ba66ed7a9f25.png)
i have two module.how they ll show in left bar

**Comment by shipanmazumder** [comment]
I have changed later and commit.

**Comment by sonu4goods** [comment]
you project is good but need some modification and testing then it ll fine.please give some time on that it ll help many developers.

**Comment by shipanmazumder** [comment]
thanks
